libass/ass.c

```diff
@@ -190,14 +192,23 @@ static int test_and_set_read_order_bit(ASS_Track *track, int id)
 {
     if (resize_read_order_bitmap(track, id) < 0)
         return -1;
-    int index = id / 32;
-    uint32_t bit = 1u << (id % 32);
+    int index = id >> 5; //div32
+    uint32_t bit = 1u << (id & 0x1F); //mod32
```

Comment:
I think those comments aren't really needed. Or at least they should be uniform between `test_and_set_read_order_bit` and `clear_read_order_bit`.